p11-kit/meson.build

```diff
@@ -140,13 +140,15 @@ if get_option('test')
 endif
 
 p11_kit_sources = [
+  'conf.c',
```

Comment:
This means that the same file `p11-kit/conf.c` is compiled twice. It would be sufficient to just link to `libp11-kit-internal.a`.